com.valvesoftware.Steam.Utility.gamescope.yml

```diff
@@ -0,0 +1,34 @@
+id: com.valvesoftware.Steam.Utility.gamescope
+default-branch: stable
+sdk: org.freedesktop.Sdk//20.08
+runtime: com.valvesoftware.Steam
+runtime-version: stable
+build-extension: true
+appstream-compose: false
+build-options:
+  prefix: /app/utils/gamescope
+  prepend-path: /app/utils/gamescope/bin
+  append-pkg-config-path: /app/utils/gamescope/lib/pkgconfig
+  strip: true
+modules:
+  - modules/libinput/libinput.json
+  - modules/xwayland/xwayland.yml
+
```

Comment:
Any reason these are split to separate files and one is in JSON?

Reply:
The `libinput.json` is mostly copied from [tv.kodi.Kodi](https://github.com/flathub/tv.kodi.Kodi/blob/master/modules/libinput/libinput.json), while the `xwayland.yml` is written from scratch.

Reply:
Ideally these should be merged into single file but this can be done after the merge.